Autopilot/AttitudeManager/OutputMixing.cpp

```diff
@@ -4,10 +4,17 @@
  * Definitions
  **********************************************************************************************************************/
 
-#define ADVERSE_YAW_MIX 0.5f // This value was taken from picPilot, it might need tuning.
-
-#define RUDDER_PROPORTION 0.75f // has to do with the angle of the inverse V tail, 0.75 means spike's tail is at 45 degrees.
-#define ELEVATOR_PROPORTION 0.75f
+// tails supported by this module
+#define CONVENTIONAL_TAIL 0
+#define INV_V_TAIL 1
+
+#ifdef SPIKE
```

Comment:
I may be wrong, but wouldn't SPIKE need to be defined in the CMakeLists.txt file? If so, I don't think you defined it.

If I am mistaken, where are you defining SPIKE?

Reply:
Not defined yet. All focus is going into Infinion for the moment. though I'm leaving this switch where it's important so that when we get to deal with Spike in a month or so, it'l be easy. The dream is to eventually be able to build with something like:

./Tools/build.bash -AIRCRAFT "SPIKE" and produce the exe tailored to that particular plane or quad or whatever instead of having to manually change source code every time.